src/imitation/algorithms/preference_comparisons.py

```diff
@@ -758,16 +765,28 @@ def _loss(
             rews2 = self._rewards(trans2)
             probs[i] = self._probability(rews1, rews2)
         # TODO(ejnnr): Here and below, > 0.5 is problematic
-        # because getting exactly 0.5 is actually somewhat
-        # common in some environments (as long as sample=False or temperature=0).
-        # In a sense that "only" creates class imbalance
-        # but it's still misleading.
+        #  because getting exactly 0.5 is actually somewhat
```

Comment:
Why the additional whitespace?

Reply:
TODO detectors in e.g. vscode and PyCharm only detect that the lines below are part of the TODO above if you indent it. Otherwise it understands it as a separate comment